ingest.py

```diff
@@ -82,17 +88,38 @@ def ingest_docs():
     # with open('docs_transformed.pkl', 'rb') as f:
     #     docs_transformed = pickle.load(f)
     
-    client = weaviate.Client(url=WEAVIATE_URL, auth_client_secret=weaviate.AuthApiKey(api_key=WEAVIATE_API_KEY))
-    client.schema.delete_class("LangChain_newest_idx") # delete the class if it already exists
+    client = weaviate.Client(
+        url=WEAVIATE_URL,
+        auth_client_secret=weaviate.AuthApiKey(api_key=WEAVIATE_API_KEY)
+    )
+    embedding = OpenAIEmbeddings(chunk_size=200)  # rate limit
+    vectorstore = Weaviate(
+        client,
+        WEAVIATE_INDEX_NAME,
+        "text",
+        embedding=embedding,
+        by_text=False,
+        attributes=["source"]
+    )
 
-    embeddings = OpenAIEmbeddings(chunk_size=200) # rate limit
+    record_manager = SQLRecordManager(
+        RECORD_MANAGER_NAMESPACE,
+        db_url=RECORD_MANAGER_DB_URL
+    )
+    record_manager.create_schema()
```

Comment:
I haven't double checked what happens if this is called twice. Semantics we want here is create if needed, but potentially may raise an exception on second create